app/src/main/java/com/android/streetworkapp/model/storage/S3StorageClient.kt

```diff
@@ -0,0 +1,100 @@
+package com.android.streetworkapp.model.storage
+
+import android.util.Log
+import aws.sdk.kotlin.runtime.auth.credentials.StaticCredentialsProvider
+import aws.sdk.kotlin.services.s3.S3Client
+import aws.sdk.kotlin.services.s3.model.ListObjectsRequest
+import aws.sdk.kotlin.services.s3.model.ObjectCannedAcl
+import aws.sdk.kotlin.services.s3.model.PutObjectRequest
+import aws.smithy.kotlin.runtime.content.ByteStream
+import aws.smithy.kotlin.runtime.net.url.Url
+import com.android.sample.BuildConfig
+
+enum class S3Clients(
+    val endpoint: String,
+    val region: String,
+    val accessKey: String,
+    val secretKey: String
+) {
+  DIGITAL_OCEAN(
+      endpoint = "https://street-work-app.fra1.digitaloceanspaces.com",
+      region = "us-east-1",
```

Comment:
the region doesn't actually matter for digitalocean

> Due to an AWS-specific behavior in all versions of the SDK except Python 3, to successfully create a new bucket, you must specify an AWS region, such as us-east-1, in your configuration. This is because, when creating a bucket, the SDK sends an entirely different payload if a custom region is specified, which results in an error.
Specifying us-east-1 does not result in slower performance, regardless of your bucket’s location. The SDK checks the region for verification purposes but never sends the payload there. Instead, it sends the payload to the specified custom endpoint.

https://docs.digitalocean.com/products/spaces/how-to/use-aws-sdks/